PriorityQueue/include/PriorityQueue.h

```diff
@@ -1,19 +1,43 @@
 #pragma once
-
-#include <iostream>
 #include <list>
-#include <exception>
+#include <stdexcept>
+
+using empty_queue_except = std::out_of_range;
 
+template <typename T>
+using priorityQueue = std::list<T>;
 
-// any templates?
+template <typename T>
+struct MyComperator {
+	T operator()(T a, T b) {
```

Comment:
Notice that marking the arguments simply as `T` means they will be copied every call to the comparator. This is bad for performance and sometimes even impossible (i.e - if `T` is or contains a `unique_ptr` or any other type that deleted the Copy-constructor). 
So in this case it definitely better to use "const-reference".
Check it out: https://www.geeksforgeeks.org/difference-between-call-by-value-and-call-by-reference/